pulp_ansible/app/galaxy/v3/serializers.py

```diff
@@ -141,6 +148,7 @@ class CollectionVersionSerializer(CollectionVersionListSerializer):
     download_url = serializers.SerializerMethodField()
 
     metadata = CollectionMetadataSerializer(source="*")
+    namespace = CollectionNamespaceSerializer(source="*")
```

Comment:
Minor: SerializerMethodField would've been more than enough.

Reply:
https://github.com/pulp/pulp_ansible/compare/master...alikins:collection_ver_5459_method is also available, but Serializer class seems more pulp style.